### PR TITLE
fix: register IPersistentLocalIdGenerator in lambda

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/AddressRegistry.Api.BackOffice.Handlers.Lambda/Function.cs [hunk 1]
```diff
@@ -1,25 +1,24 @@
 using Amazon.Lambda.Core;
+using Amazon.Lambda.Serialization.Json;
 
-[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.Json.JsonSerializer))]
+[assembly: LambdaSerializer(typeof(JsonSerializer))]
```
Comment (lvermeulen) [inline src/AddressRegistry.Api.BackOffice.Handlers.Lambda/Function.cs:4]: for clarity please leave the entire namespace in this line

**Review by lvermeulen — CHANGES_REQUESTED** [review]
leave namespace in line

File: src/AddressRegistry.Api.BackOffice.Handlers.Lambda/Function.cs (hunk 1 above)
Comment (jvandaal) [reply src/AddressRegistry.Api.BackOffice.Handlers.Lambda/Function.cs]: I've put it back. Rider shortcut changed it.

**Review by lvermeulen — APPROVED** [review]

**Pull request merged**